packages/odyssey-storybook/package.json

```diff
@@ -31,11 +31,13 @@
     "@storybook/core-events": "^6.4.22",
     "@storybook/react": "^6.4.22",
     "@storybook/theming": "^6.4.22",
+    "@testing-library/testcafe": "^4.4.1",
     "@types/mdx": "^2.0.1",
     "@types/react": "^17.0.30",
     "babel-loader": "^8",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "testcafe": "https://github.com/okta/testcafe.git",
```

Comment:
I'm concerned about using this version of testcafe as we haven't updated it in years. Why aren't we using the public version?

Reply:
I have no objections to that. It was actually one of my questions. This specific source, I grabbed from the package.json of the admin-nav package.